mermaid_handler.py

```
@@ -0,0 +1,295 @@
+                payload = {
+                    "diagram_source": mermaid_code,
+                    "diagram_type": "mermaid",
+                    "output_format": "png"
+                }
+                
+                async with aiohttp.ClientSession() as session:
+                    async with session.post(
+                        self.kroki_post_url,
+                        json=payload,
+                        timeout=10
+                    ) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)